velox/exec/AggregateInfo.cpp

```diff
@@ -80,7 +77,7 @@ std::vector<AggregateInfo> toAggregateInfo(
       } else {
         VELOX_FAIL(
             "Expression must be field access, constant, or "
-            "lambda (HashAggregation): {}",
+            "lambda (HashAggregation/StreamingAggregation): {}",
```

Comment:
Can we drop (HashAggregation/StreamingAggregation) altogether? Any particular need to include it in the error message?

Reply:
Yes, I will drop them.